gui/mutt_window.c

```diff
@@ -808,3 +809,73 @@ bool window_status_on_top(struct MuttWindow *panel, struct ConfigSubset *sub)
   window_invalidate_all();
   return true;
 }
+
+/**
+ * mutt_window_is_child - Check whether a window is a child of a parent
+ * @param parent Parent Window
+ * @param win    Window
+ * @retval true  Window is a child of the parent
+ * @retval false Window is not a child of the parent
+ */
+static bool mutt_window_is_child(struct MuttWindow *parent, struct MuttWindow *win)
```

Comment:
We could probably get away without this.
All the children get added by `mutt_window_add_child()` which sets `MuttWindow.parent`.
Checking `win1.parent == win2.parent` should be enough.

Reply:
Ah, I didn't even think of `win.parent`. I've remove `mutt_is_child` now.